scripts/openmrs/frontend_assembly/build-openmrs-frontend.groovy

```diff
@@ -38,6 +43,7 @@ if (shouldBuildFrontend) {
     }
 
     log.info("Running assemble command...")
+    log.info(assembleCommand)
```

Comment:
Do we need this?

Reply:
I think it's useful to show the assemble command being run